### ISSUE TITLE
Uldaman stone keeper no animation for nova

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#4400]
> Uldaman stone keeper no animation for nova

.go xyz 112.643000 277.374000 -26.532200 70 0.331525

**Comment by Shadownflame** [comment]
Apparently the Altar of the Keepers can be used by a single person. This thing:
![WoWScrnShot_102724_200225](https://github.com/user-attachments/assets/08ab2773-704f-4bb7-8511-2fb40277fa54)

I don't remember if the [Self Destruct ](https://www.wowhead.com/tbc/spell=9874/self-destruct) spell had an animation on retail but it would make sense if it did. I've never seen them use their aoe knockdown (trample), might need more than 1 person to test that.

**Comment by kaytotes** [comment]
Altar of Keepers should require 3 so I'll fix that. Note, on internal realm I will keep it at 1 so that won't be testable.

As for the spell, blizzlike spell, blizzlike data. Impossible for it to mismatch. I will check vMangos to see if there is possibly a linked visual aura that's missing here but if I don't find one I'll be considering that blizzlike.

**Comment by T1ti** [comment]
Not sure if the issue is that they don't do an animation before triggering the nova, or if there's no nova animation at all, but on official classic it looks like that https://youtu.be/pQ7zy_ix1k0?feature=shared&t=1891

**Comment by Shadownflame** [comment]
I believe the chain reaction script is simply missing altogether. If I'm doing the event with a high level character (who doesn't pull the activating keepers via proximity aggro) then the keepers don't auto engage at all. 
When a keeper is close to dying, or dies (depends on DPS speed), two things should certainly happen:
1) activate AND engage the next keeper
2) die if not yet dead, then blast people's health off with the explosion 

Only the second one is currently happening and it's entirely possible that the graphic nova visual belongs to the first action instead.
PS: I'm not even sure if the second one works properly because the mechanism doesn't seem to be able to trigger at low health on epoch, only post-mortem. The keepers can sit eternally on 40/9000 health without engaging the next golem and self-destructing ,which is not how the self-destruct spell is described to work, and is not what we see on the videos either.

**Comment by T1ti** [comment]
Pretty sure it always happens when they're super low regardless of DPS, they melt them down in his vid but they still all do the explosion https://youtu.be/GHsytWhHKNA?feature=shared&t=2022

**Comment by Shadownflame** [comment]
Yes, it certainly should. Once you get them to a certain health, the mob is supposed to go into self-destruct mode. That's the entire point of self-destruct.
However, what I'm saying is that on Epoch it won't trigger at low health like it does on classic videos. It always happens only after the mob is fully killed by your DPS, no matter how slow your DPS is. An irrelevant detail at first glance, but it showcases how differently the sequence is scripted on retail. 
On classic there is a clearly distinguishable state when the mob is low health but still alive > casts animation > triggers next mob to engage > the first mob drops dead and the group takes AOE damage.
On epoch the low health trigger is entirely missing and only when your group DPS executes the mob you take damage and the next mob's health bar pops up simultaneously (without engaging).

**Comment by v-mstrs** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/15744 maybe not 1 to 1
